File: Santiago Sanchez/Generics/src/generics/Customer.java
package generics;



public class Customer {
    
    Customer(){
        
    }

    public int id;
    public String name, email;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void Data_Return() {
        System.out.println("Email: " + email + "   Nombre:" + name);
    }
    
    public Customer(String email, String nombre) {
       
        this.email = email;
        this.name = nombre;

    }
    

}

File: Moises Dominguez/Generics/Generics/Customer.cs
using System.Collections.Generic;

namespace Generics
{
    class Customer
    {
        public int customerId { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public Stack<Order> Orders { get; set; }
        public Customer()
        {
            Orders = new Stack<Order>();
        }
    }
}
